distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/ErrorPumpThread.java

```diff
@@ -68,6 +69,9 @@ void drain() {
         nonInterruptibleVoid(this::join);
     }
 
+    /** List of messages / lines to filter from the output. */
+    List<String> filter = List.of("WARNING: Using incubator modules: jdk.incubator.vector");
```

Comment:
nit: can this be moved to the top of the class and made `private static final`?

Reply:
Sorry, only seen this after merging. I'll move this as a follow-up.